python-backend/app/scheduled_jobs/ETL_jobs.py

```diff
@@ -11,7 +11,7 @@ def _schedule_ETL_jobs(app):
 @register_apm
 def _run_ETL():
     with sched.app.app_context():
-        job_running = cache.get(ETL())
+        job_running = cache.get(ETL)
```

Comment:
Was this change intended? Does it have an effect?

Reply:
@JustinMacaulay  requested this change, ETL isn't a function... so that would have been bad, it is unrelated to my changes.